tests/storm/storm_client.py

```diff
@@ -0,0 +1,47 @@
+import click
```

Comment:
`click` does not exist in our `requirements` nor `extra-requirements.txt`

Reply:
can u turn them into `pytest` tests with proper parametrization? @numb3r3

Reply:
@JoanFM sure, i will convert this as pytest cases